webpack/InsightsCloudSync/Components/RemediationModal/RemediationModalFooter.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { JOB_INVOCATION_PATH } from './RemediationTableConstants';
+
+const ModalFooter = ({ toggleModal, resolutions }) => {
+  let token = document.querySelector('meta[name="csrf-token"]');
+  token = token?.content || '';
+  return (
+    <form action={JOB_INVOCATION_PATH} method="post">
+      <Button type="submit" key="confirm" variant="primary">
+        {__('Remediate')}
+      </Button>
+      <Button key="cancel" variant="link" onClick={toggleModal}>
+        {__('Cancel')}
+      </Button>
+      <input type="hidden" name="feature" value="insights_remediation" />
+      <input type="hidden" name="authenticity_token" value={token} />
+      {resolutions.map((res, i) => (
+        <React.Fragment key={i}>
+          <input
+            type="hidden"
+            name="resolutions[]"
+            key={res.id}
+            value={JSON.stringify(res)}
+          />
+        </React.Fragment>
+      ))}
```

Comment:
Instead of multiple `<input>` fields, we should pass a single string with json value of hit_id and remediation_id pairs.
something like:

``` js
{
//assuming resolutions is an array of {hit_id, resolution_id} pairs:
          <input
            type="hidden"
            name="inputs[hit_remediation_pairs]"
            value={JSON.stringify(resolutions)}
           />
}
```

Notice the name of the input, it's the name of the parameter of the job template.

Reply:
Nice! that works :D